x/posts/client/cli/tx.go

```diff
@@ -347,3 +348,52 @@ Then, the "Cat" answer has index 0 and and the "Dog" answer has index 1.
 
 	return cmd
 }
+
+// GetCmdMovePost returns the command allowing to move an existing post to another subspace
+func GetCmdMovePost() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "move-post [subspace-id] [post-id] [target-subspace-id] [target-section-id]",
+		Args:    cobra.ExactArgs(4),
+		Short:   "Move an existing post to another subspace",
+		Example: fmt.Sprintf(`%s tx posts move-post 1 1 2 1 --from alice`, version.AppName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			subspaceID, err := subspacestypes.ParseSubspaceID(args[0])
+			if err != nil {
+				return err
+			}
+
+			postID, err := types.ParsePostID(args[1])
+			if err != nil {
+				return err
+			}
+
+			targetSubspaceID, err := subspacestypes.ParseSubspaceID(args[2])
+			if err != nil {
+				return err
+			}
+
+			targetSectionID, err := subspacestypes.ParseSectionID(args[2])
```

Comment:
This should be `args[3]`
```suggestion
			targetSectionID, err := subspacestypes.ParseSectionID(args[3])
```